prow/cluster/jobs/istio/tools/istio.tools.yaml

```diff
@@ -90,8 +90,7 @@ postsubmits:
         - ^master$
       run_if_changed: '^docker/build-tools/'
       annotations:
-        testgrid-dashboards: "tools"
-        testgrid-tab-name: "build-tools"
```

Comment:
Did you mean to remove the tab name?

Reply:
yes, this was just added today so it's not on the testgrid at all. so far our pattern has been to not set this so everything is consistent with the test name and easier to find